spec/requests/api/v0/followings_spec.rb

```diff
@@ -20,6 +20,19 @@
       end
     end
 
+    # See: https://github.com/thepracticaldev/dev.to/issues/6119
+    context "when user is authorized with an API key" do
+      let_it_be(:api_secret) { create(:api_secret, user: user) }
+
+      %w[users tags organizations podcasts].each do |type|
```

Comment:
All the methods used `current_user`, so I added a spec for each case. Garbage objects are IMHO less of a concern so I left the array inline, which also helps with Rubocop not complaining about `stub_const` (it can't now that we're not trying to do that but instead define a constant for use inside the spec, due to the way constant lookup works in Ruby).